conjur-cli.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |gem|
   gem.executables -= %w{parse-changelog.sh}
 
   gem.add_dependency 'activesupport', '>= 4.2'
-  gem.add_dependency 'conjur-api', '~> 5.3'
+  gem.add_dependency 'conjur-api', '~> 5.3.6'
```

Comment:
I don't think that we need to change behavior there

> The specifier `~>` has a special meaning, best shown by example. `~> 2.0.3` is identical to `>= 2.0.3` and `< 2.1`. `~> 2.1` is identical to `>= 2.1` and `< 3.0`.

So in our case adding `.6` changes the condition focus from major version part to minor one.